templates/taxi/manufacturer_delete_confirm.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete manufacturer</h1>
+  <p>Are you sure wou want to delete the manufacturer: {{ manufacturer }}</p>
+  <p><i>All cars with this manufacturer will be also deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+  <input class="btn btn-danger" type="submit" value="Delete">
```

Comment:
Add a `Cancel` button along with a `Delete` one to return to the previous page and cancel the deletion as mentioned in the checklist. Apply it to all delete forms